browser/src/control/Control.StatusBar.js

```diff
@@ -257,13 +261,13 @@ class StatusBar extends JSDialog.Toolbar {
 			{type: 'separator', id: 'searchbreak', orientation: 'vertical'},
 			this._generateHtmlItem('statusdocpos'), 					// spreadsheet
 			this._generateHtmlItem('rowcolselcount', 1), 					// spreadsheet
-			this._generateHtmlItem('statepagenumber'), 					// text
+			this._generateHtmlItem('statepagenumber', undefined, 'gotopage'), 		// text
 			this._generateHtmlItem('statewordcount', 1), 					// text
 			this._generateHtmlItem('insertmode', 5),						// spreadsheet, text
 			this._generateHtmlItem('showcomments', 4),					    // text
 			this._generateHtmlItem('statusselectionmode', 6),				// text
-			this._generateHtmlItem('slidestatus'),						// presentation
-			this._generateHtmlItem('pagestatus'),						// drawing
+			this._generateHtmlItem('slidestatus', undefined, 'gotopage'),			// presentation
+			this._generateHtmlItem('pagestatus', undefined, 'gotopage'),			// drawing
```

Comment:
hmm, I want to at some point remove this HTMLCONTENT widget. Would be good to not allow for more extending :)

Can't we just setup onclick in the implementation of particular widgets?
https://github.com/CollaboraOnline/online/blob/master/browser/src/control/jsdialog/Widget.HTMLContent.ts#L69

I think it will be less code and less distributed over codebase

Reply:
done